test/test_helper.exs

```diff
@@ -48,4 +48,11 @@ defmodule Protobuf.TestHelpers do
     |> Code.Typespec.type_to_quoted()
     |> Macro.to_string()
   end
+
+  # This code is taken from Code.fetch_docs/1 in Elixir (v1.13 in particular).
+  def fetch_docs_from_bytecode(module, bytecode) when is_atom(module) and is_binary(bytecode) do
```

Comment:
This function exists because `Code.fetch_docs/1` only supports modules coming from BEAM files as an argument - which is not the case for modules coming from `Code.compile_file/1`.